utils/release_helper.py

```diff
@@ -0,0 +1,790 @@
+        if branch_to_remove:
+            remove_old_stabilization_branch(branch_to_remove)
+        else:
+            print('Great! There is no branch to be removed.')
+
+
+def release_prep(repo, args) -> None:
+    if args.contributors:
+        update_contributors()
+
+    stab_branch_name = get_next_stabilization_branch(repo)
+    if args.branch:
+        print('git checkout master\n'
+              'git pull upstream master\n'
+              f'git checkout -b {stab_branch_name}\n'
+              f'git push -u upstream {stab_branch_name}')
+
+    if args.milestone:
+        if is_next_release_in_progress(repo):
+            stab_milestone = get_latest_version_milestone(repo)
+            next_release_version = get_next_release_version(repo)
+            new_milestone = bump_version(next_release_version, 'minor')
+            close_milestone(stab_milestone)
+            create_repo_milestone(repo, new_milestone)
+        else:
+            print(f'Milestones can be managed after the "{stab_branch_name}" branch is created.')
+
+    if args.issues:
+        issues_list = get_open_issues_with_milestone(repo)
+        update_milestone_in_issues(repo, issues_list)
+
+    if args.prs:
+        prs_list = get_open_prs_with_milestone(repo)
+        update_milestone_in_prs(repo, prs_list)
+
+    if args.bump_version:
+        bump_master_version(repo)
+
+
+def release(repo, args) -> None:
+    if not is_next_release_in_progress(repo):
+        if args.tag:
+            next_version = get_next_release_version(repo)
+            tag_name = f'v{next_version}'
+            stab_branch = get_next_stabilization_branch(repo)
+            print('Release process starts when a version tag is added to the branch.\n'
+                  'It is better to do it manually. But here are the commands you need:\n')
+            print(f'git tag {tag_name} {stab_branch}\n'
+                  'git push --tags')
+
+        if args.message:
+            message = get_release_start_message(repo)
+            print_communication_channels('release')
+            print(message)
+    else:
+        print('Please, ensure that all steps in the Release Preparation are concluded.')
+
+
+def finish(repo, args) -> None:
+    if not is_next_release_in_progress(repo):
+        if args.message:
+            message = get_release_end_message(repo)
+            print_communication_channels('finish')
+            print(message)
+    else:
+        print('Please, ensure that all steps in the Release process are concluded.')
+
+
+def parse_arguments():
+    '''Call argparse to process input parameters and return parsed args'''
+    parser = argparse.ArgumentParser(
+        description="Automate Github tasks included in the Release Process.",
+        epilog="Example: release_helper.py -c ~/secrets.ini -r ComplianceAsCode/content stats",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-c', '--creds-file', default='~/secret.ini',
```

Comment:
The default ~/secret.ini does not work as expected. I think you have to absolutize the path before you hand it to the create_github_session function.

Reply:
Hum, good point. I will update the default value.

Reply:
Actually fixed the function to expand `~`.
Fixed in https://github.com/ComplianceAsCode/content/pull/10076/commits/82e5fcd2553b8de977391e4e0bf370dac09a8154

Thanks for the catch @vojtapolasek